sdk/inc/azure/iot/az_iot_hub_client.h

```diff
@@ -712,12 +712,15 @@ typedef enum
 {
   AZ_IOT_HUB_CLIENT_PROPERTIES_RESPONSE_TYPE_GET
   = AZ_IOT_HUB_CLIENT_TWIN_RESPONSE_TYPE_GET, /**< A response from a properties "GET" request. */
-  AZ_IOT_HUB_CLIENT_PROPERTIES_RESPONSE_TYPE_DESIRED_PROPERTIES
+  AZ_IOT_HUB_CLIENT_PROPERTIES_RESPONSE_TYPE_WRITEABLE
   = AZ_IOT_HUB_CLIENT_TWIN_RESPONSE_TYPE_DESIRED_PROPERTIES, /**< A "PATCH" response with a payload
```

Comment:
I still think calling this a `response` type is funny.  It's not a response from the Hub, but rather a message.  
Why don't we include PATCH in the name, like we do above with GET?  TYPE_PATCH_REQUEST ?

I'm glad you're looking into this because I found the naming curious as well.  I have also been playing around with it.